src/quicktimevideo.cpp

```diff
@@ -1155,9 +1155,12 @@ void QuickTimeVideo::timeToSampleDecoder() {
     io_->readOrThrow(buf.data(), 4);
     timeOfFrames = Safe::add(timeOfFrames, temp * buf.read_uint32(0, bigEndian));
   }
-  if (currentStream_ == Video)
+  if (currentStream_ == Video) {
+    if (timeOfFrames == 0)
+      timeOfFrames = 1;
```

Comment:
@neheb: Did coverity complain about a possible divide-by-zero? I'm not convinced it's a bug. We're using floating point arithmetic here, so it wouldn't cause a crash - it would just compute Inf or NaN.

Reply:
Yeah that's it. I'm not convinced it can be 0 based on the above code.